src/Ydb.Sdk/src/Services/Table/SessionPool.cs

```diff
@@ -83,6 +91,21 @@ public SessionPool(Driver driver, SessionPoolConfig config)
     }
 
     public async Task<GetSessionResponse> GetSession()
+    {
+        const int maxAttempts = 100;
+
+        GetSessionResponse getSessionResponse = null!;
+        for (var attempt = 0; attempt < maxAttempts; attempt++)
```

Comment:
Better to use timeout instead of attempts and need pause between attemps. it need some time to repair the situation (create session, return idle session to the pool, ...)